### PR TITLE
Murloc stuff

### PR DESCRIPTION
**Pull request by Myphicbowser**
## Changelog:
- Added the Living Divinity Tenet, a tenet based on sucking up to your Liege and other living and natural things.
- Changed Murloc religious tenets entirely.
- Modified the Murloc Culture, replacing Frugal Armorsmithes with Adaptative Skirmishers
- Removed Murlocs diverging from Gorlocs, as the lore isn't clear and a few discussions haven't really proven a point either way

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

### CODE REVIEW

File: common/religion/religions/water_group.txt [hunk 1]
```diff
@@ -334,9 +334,17 @@
 			#Main Group
 			doctrine = doctrine_no_head
 
-			doctrine = tenet_warmonger
-			doctrine = tenet_bloody_feast
-			doctrine = tenet_communal_identity
+			#Set like this for Natural Primitivism
```
Comment (ElMariuso) [inline common/religion/religions/water_group.txt:337]: Uh ? Explain why, I don't get it. Maybe it's something I'm not aware of

File: common/religion/religions/water_group.txt [hunk 2]
```diff
@@ -334,9 +334,17 @@
...
+			doctrine:doctrine_adultery_women_accepted
+			doctrine:doctrine_homosexuality_accepted
+			doctrine:doctrine_adultery_men_accepted
+			doctrine:doctrine_witchcraft_accepted
+			doctrine:doctrine_deviancy_accepted
+			doctrine:doctrine_kinslaying_accepted
+
+			doctrine = tenet_natural_primitivism
```
Comment (ElMariuso) [inline common/religion/religions/water_group.txt:345]: That's not really religious

File: common/religion/religions/water_group.txt [hunk 3]
```diff
@@ -334,9 +334,17 @@
...
+			doctrine = tenet_ritual_celebrations
```
Comment (ElMariuso) [inline common/religion/religions/water_group.txt:346]: Why ?

File: common/religion/doctrines/00_core_tenets.txt [hunk 4]
```diff
@@ -3896,4 +3901,42 @@
 			cannibalism_legal = yes
 		}
 	}
-}
+
+	#Living Divinity, basically Warrior Monks and Unrelenting Faith
+	tenet_living_divinity = {
+		icon = wc_core_tenet_flexing_muscles
```
Comment (ElMariuso) [inline common/religion/doctrines/00_core_tenets.txt:3907]: Find maybe something else

File: common/religion/doctrines/00_core_tenets.txt [hunk 5]
```diff
@@ -3896,4 +3901,42 @@
...
+			value = faith_tenet_cost_low
+
+			if = {
+				limit = { has_doctrine = tenet_living_divinity }
+				multiply = faith_unchanged_doctrine_cost_mult
+			}
+		}
+		
+		is_shown = {
+			religion = { NOT = { is_in_family = rf_light } }
```
Comment (ElMariuso) [inline common/religion/doctrines/00_core_tenets.txt:3918]: Not sure it work, does it ? What about error.log ?

**Review by ElMariuso — COMMENTED** [review]
I can't really test that in game 'cause it's on your fork sadly, but here is some comments

**Comment by Spectral-Baron** [comment]
- mod did not exceed 5.5 GB in task manager
- maybe swap wrathful being sinful and calm being virtuous? murlocs seem wrathful and very rarely calm, would grant some Martial + Dread too; seems fitting
- saw a few errors in error.log (see below)
```
[21:15:04][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: doctrine:doctrine_adultery_women_accepted, near line: 339" in file: "common/religion/religions/water_group.txt" near line: 339
[21:15:04][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: doctrine:doctrine_adultery_men_accepted, near line: 341" in file: "common/religion/religions/water_group.txt" near line: 341
[21:15:04][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: doctrine:doctrine_deviancy_accepted, near line: 343" in file: "common/religion/religions/water_group.txt" near line: 343
```

**Comment by Raged50** [comment]
"Removed Murlocs diverging from Gorlocs, as the lore isn't clear and a few discussions haven't really proven a point either way"

needs to be discussed in ck3-dev channel before I sign off on this being implemented, either start a thread or talk about it in the general chat and see what everyone's input is

File: common/religion/doctrines/00_core_tenets.txt (hunk 4 above)
Comment (Myphicbowser) [reply common/religion/doctrines/00_core_tenets.txt:3907]: I did ask for something in the 2d art channel, just kinda waiting on that

File: localization/english/religion/wc_core_tenets_l_english.yml [hunk 6]
```diff
@@ -44,6 +44,9 @@
  tenet_cult_of_strength_name:0 "Cult of Strength"
  tenet_cult_of_strength_desc:0 "The stronger we are, the closer we are to divinity. Never stop fighting, never stop growing. Each day, stronger than before."
 
+ tenet_living_divinity_name:0 "Living Divinity"
+ tenet_living_divinity_desc:0 "The Divine not only walks among us, but they lead as well. To fight in their name and march under their banner is a blessing few know."
```
Comment (edmund-h) [inline localization/english/religion/wc_core_tenets_l_english.yml:48]:
```suggestion
 tenet_living_divinity_desc:0 "The Divine not only walks among us, but leads our people as well. To fight in their name and march under their banner is a blessing worth giving one’s life for."
```

**Pull request closed without merging**